internal/types/document.go

```diff
@@ -23,6 +23,11 @@ import (
 	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
+const (
+	// The idKey is key for _id field.
+	idKey = "_id"
```

Comment:
tbh, I never understood constants like this one, especially given the fact that we have "_id" string across many files in many packages

Reply:
I introduce that constant just because of the linter warning.

Reply:
In the next PR, let's remove that constant and tweak linter settings instead. We could add "_id" string to exceptions, for example

Reply:
@w84thesun Please ack, resolve conversation, and merge PR